docs/source/examples/mitiq-paper/mitiq-paper-codeblocks.md

```diff
@@ -1,64 +1,51 @@
 ---
-jupytext:
-  text_representation:
-    extension: .md
-    format_name: myst
-    format_version: 0.13
-    jupytext_version: 1.11.1
-kernelspec:
-  display_name: Python 3
-  language: python
-  name: python3
----
-
-+++ {"id": "wjnIQWX8d5UY"}
-
+jupyter:
+  jupytext:
+    text_representation:
+      extension: .md
+      format_name: markdown
+      format_version: '1.3'
+      jupytext_version: 1.16.1
+  kernelspec:
+    display_name: Python 3
+    language: python
+    name: python3
+---
+
+<!-- #region id="wjnIQWX8d5UY" -->
 # Mitiq paper codeblocks
 
 Codeblocks from the main text of the [Mitiq whitepaper](https://quantum-journal.org/papers/q-2022-08-11-774/) {cite}`LaRose_2022_Quantum` published in [Quantum](https://quantum-journal.org/).
+<!-- #endregion -->
 
-+++ {"id": "DRplhovffVfk"}
-
+<!-- #region id="DRplhovffVfk" -->
```

Comment:
I wonder why there is so much noise in the diff of this file. It would be nice to have only a diff that matters.

Reply:
I'm not an expert in Myst, but perhaps it's because I converted this to a jupyter notebook to run it and then converted it back to .md using jupytext? @natestemen Do you have insight on this?

Reply:
Correct me if I'm wrong here, but it looks like the .md version of [this notebook in main](https://github.com/unitaryfund/mitiq/blob/main/docs/source/examples/mitiq-paper/mitiq-paper-codeblocks.md) is actually also messed up?

Reply:
<img width="1373" alt="Screen Shot 2024-04-17 at 1 24 07 PM" src="https://github.com/unitaryfund/mitiq/assets/15827191/707d9603-df7a-4a16-9a61-91830b0e0eb0">

Reply:
<img width="399" alt="Screen Shot 2024-04-17 at 1 35 44 PM" src="https://github.com/unitaryfund/mitiq/assets/15827191/781e4269-de86-4cba-b08f-be71e1f3d793">
Also has collab info. Do we want all this present?

Reply:
@natestemen This item should be the last thing. Not sure if we want to include fixing the formatting on this notebook in this PR or not, since it's present in main. The functional part is done.

Reply:
It looks like you've done a bit of clean up in this PR, but lets keep the rest in a subsequent PR. Even though the markdown does render a little strangely here on GitHub, it looks fine on the [live documentation](https://mitiq.readthedocs.io/en/stable/examples/mitiq-paper/mitiq-paper-codeblocks.html), which is what matters most.

Reply:
Cool. In that case should be ready to merge.

Reply:
A formal approval is needed before merging. Now that you've added more commits since the first review, typically a second round of review would be initiated to ensure the new additions look good as well. To initiate another round of review you can request it using the "Re-request review" feature.